src/main/java/util/AllureUtil.java

```diff
@@ -0,0 +1,58 @@
+package util;
+
+import com.automation.remarks.video.recorder.VideoRecorder;
+import drivers.DriverManager;
+import io.qameta.allure.Attachment;
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.testng.ITestResult;
+import java.io.File;
+import java.io.IOException;
+import static util.EncoderUtil.convertAviToMp4;
+
+public class AllureUtil {
+
+    /***
+     Attach log file to allure report
+     */
+    @Attachment(value = "Test.log", type = "text/plain")
+    public static byte[] appendLogToAllure() {
+        try {
+            return FileUtils.readFileToByteArray(new File("target/test.log"));
```

Comment:
Store filename and filepath in a separate constant as it might be used several times here

Reply:
noted